aea/crypto/base.py

```diff
@@ -67,7 +68,11 @@ def public_key(self) -> str:
 
         :return: a public key string in base58 format
         """
-        return self._public_key_b58
+        fetch_crypt = FetchCrypto()
+        public_keys = {}
```

Comment:
Would use typing: `public_keys = {}  # type: Dict[str, str]`.

Reply:
Removed from this file.